src/test/store/receive-profile.test.js

```diff
@@ -1868,7 +1883,8 @@ describe('actions/receive-profile', function() {
           .fn()
           .mockRejectedValue(new Error('No symbol tables available')),
       };
-      window.geckoProfilerPromise = Promise.resolve(geckoProfiler);
+
+      simulateOldWebChannelAndFrameScript(geckoProfiler);
 
       simulateSymbolStoreHasNoCache();
 
```

Comment:
Am I right that there's no new test for the new web channel implementation?

Reply:
Can you also please add tests to web-channel.test.js covering the case of an old Firefox version (where `_fixupOldResponseMessageIfNeeded` would be called especially)?